Server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
+
+app.get('/:campsiteId', (req, res) => {
+  console.log(req.params)
```

Comment:
@alefranco1 @bastipacudan Try not to leave stray console logs in code that will be merged into master.